### ISSUE TITLE
Sometimes resizing stops working

### ISSUE THREAD

**Issue by benbarbour** [Oromis/foundry-combat-focus#10]
Foundry 0.8.6, dnd5e, Chrome browser.

I could resize the split when I first loaded, but fairly consistently it seems to stop working. I haven't been able to determine what's triggering it... I just play a while and then suddenly it stops working. The cursor still changes to a resize icon when I hover over the split, but I can't drag it.

I don't see any relevant looking warnings or errors in the console, sorry.

**Comment by Oromis** [comment]
Thanks for the report! I'll try to reproduce that, but no luck so far. What kind of timeframe are we talking about?

**Comment by benbarbour** [comment]
It's pretty frequent for me, usually within the first few minutes of
messing around with foundry.

Later tonight I'll try and come up with a more concrete set of reproduction
steps, since you can't seem to trigger it. I'll try it without other
modules too... maybe there's some kind of conflict.

**Comment by benbarbour** [comment]
Actually, right now it's happening quite consistently with just Combat Focus enabled, and it right from load. Updated to 0.8.7.

The "onStartDrag" function is never called, no matter where I click. "registerDragResizeListener()" is called though.

If I put a breakpoint in registerDragResizeListener and lookup the element with id "chat", it returns a "template" element. If I wait until the page is fully loaded (which it _should_ be here, because that's what the "ready" hook is for, right?) and then do a `document.getElementById('chat')` again, then I get a "section" element. I think the listener is being added to the template, but not the section. :/ I tried changing your ready hook handler to call all those functions after a 1.5 second timeout, and that seems to work, though it's obviously a hack. It would be best to be able to hook into something guaranteed to fire after those templates are filled.

**Comment by benbarbour** [comment]
```
Hooks.on('renderSidebarTab', function() {
  if ($('section#chat.tab').length) {
    injectSidebarHook()
    registerDragResizeListener()
    updateCombatTrackerStyle()
    updateStyleElement()
  }
})
```

seems to do the trick, although I'm not too familiar with all this. Probably needs a bit more testing.

**Comment by Oromis** [comment]
Thanks for the research! I'm still wondering why it's not an issue for me... Probably a race condition and your machine is either faster or slower than mine. I'll try to fix it :)

**Comment by Oromis** [comment]
Fixed in v1.4.1 :)

Thanks for your help with this!
